client/src/components/common/Table/Table.css

```diff
@@ -1,10 +1,76 @@
-.table {
+
+body .table {
+  padding-top: 0%;
+  padding-left: 8vw;
+  padding-right: 8vW;
   display: flex;
   flex-direction: column;
 }
 
-.header {
+.headerRow {
+  color: white;
+  display: flex;
+  /*border-bottom: 2px solid black; */
+}
+
+.headerColumn {
+  width: 50vw;
+  color: white;
+  display: flex;
+  flex-direction: row;
+  background-color:#a26e49;
+  text-align: center;
+  justify-content: flex-start;
+  padding-top: 0.5vw;
+  padding-left: 8vw;
+  padding-right: 1vW;
+  padding-bottom: 0.5vw;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.row {
   display: flex;
-  border: 1px solid black;
-  padding: 1rem; 
-}
\ No newline at end of file
+  color: #22353c;
+}
+
+.row:nth-child(even) {
+  background-color:#f9f0e7;
+}
+.row:nth-child(odd) {
+  background-color:#e7e3e2;
+}
+.column {
+  display: flex;
+  justify-content: flex-start;
+  width: 50vw;
+  text-align: center;
+  border-bottom: 1px solid #b8a49b;
+  font-size: 15px;
+  padding-top: 0.5vw;
+  padding-left: 8vw;
+  padding-right: 1vW;
+  padding-bottom: 0.5vw;
```

Comment:
let's remember on the future change the padding to be in one line. Padding css property receives 4 params, please read the documentation about this